scripts/abbreviations/LocalizeAbbreviations.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,272 @@
+// !$*UTF8*$!
```

Comment:
Could we fold this project into the main MapboxNavigation project? Keeping everything in one place could be more convenient for making local changes. (The SwiftCLI dependency would go in Cartfile.private to avoid interfering with normal developer workflows.)

Or is the idea to keep the abbreviation project separate so that we can easily split it into its own repository in the future?